docs/guides/basics/parameters/index.md

```diff
@@ -243,6 +304,38 @@ coda.makeParameter({
 ```
 
 
+## Recommended parameter types
+
+The table below shows the recommended parameter type to use with various types of Coda columns and values.
+
+| Type          | Supported | Recommended   | Notes                                                             |
+| ------------- | --------- | ------------- | ----------------------------------------------------------------- |
+| Text          | ✅ Yes    | `String`      | Use `Html` if the formatting is important.                        |
+| Select list   | ✅ Yes    | `StringArray` | Works for both single and multi-value select lists.               |
+| Number        | ✅ Yes    | `Number`      |                                                                   |
+| Percent       | ✅ Yes    | `Number`      | Passed as a fraction.                                             |
+| Currency      | ✅ Yes    | `Number`      | Use `String` to get currency symbol.                              |
+| Slider        | ✅ Yes    | `Number`      |                                                                   |
+| Scale         | ✅ Yes    | `Number`      |                                                                   |
+| Date          | ✅ Yes    | `String`      | See the [Dates & times][dates] guide.                             |
+| Time          | ✅ Yes    | `Number`      | See the [Dates & times][dates] guide.                             |
+| Date and time | ✅ Yes    | `Date`        | See the [Dates & times][dates] guide.                             |
+| Duration      | ✅ Yes    | `Number`      | See the [Dates & times][dates] guide.                             |
+| Checkbox      | ✅ Yes    | `Boolean`     |                                                                   |
+| People        | ❌ No     |               | Use `String` to get the person's name.                            |
+| Reaction      | ❌ No     |               | Use `StringArray` to get the names of the people that reacted.    |
+| Button        | ❌ No     |               |                                                                   |
+| Image         | ✅ Yes    | `ImageArray`  | Image column can contain multiple images.                         |
+| Image URL     | ✅ Yes    | `Image`       |                                                                   |
+| File          | ❌ No     |               | Using `Image` will work, but show an error in the formula editor. |
+| Lookup        | ❌ No     |               | Use `String` to get the display name of the row.                  |
```

Comment:
You may need StringArray if the Lookup is a multiselect.

Reply:
Done